pkg/utils/utils.go

```diff
@@ -41,3 +41,7 @@ func MergeMaps(parent, child map[string]string) map[string]string {
 	}
 	return parent
 }
+
+func PackageID(name, version string) string {
```

Comment:
Just FYI: We wanted to change how to generate package IDs depending on ecosystem. For example, Java will probably create an ID like `com.google.code.gson:gson`. That's why we kept `ID()` in each package. But we can refactor it later. Thanks.